packages/cloud_controller_ng/pre_packaging

```diff
@@ -2,15 +2,28 @@ set -e -x
 
 cd ${BUILD_DIR}/cloud_controller_ng
 
+input_file="Gemfile"
+
+if [[ ! -f "$input_file" ]]; then
+  echo "Error: $input_file does not exist."
+  exit 1
+fi
+
+file_content=$(<"$input_file")
+modified_content=$(echo "$file_content" | sed '/^group :development do/,/^end/ d' | sed '/^group :test do/,/^end/ d')
+echo "$modified_content" > "$input_file"
+
 rm -rf ./vendor/cache/
 
 set +e
```

Comment:
As you have moved the `set +e` / `set -e` around the possibly failing command, the "outer" `set +e` / `set -e` have to be removed (here + line 33).